websites/management/commands/fix_file_urls.py

```diff
@@ -0,0 +1,127 @@
+""" Fix WebsiteContent files that are missing the Website name in their paths"""
+import csv
+import re
+from typing import Dict, List
+
+import boto3
+from django.conf import settings
+from django.core.management import BaseCommand
+from mitol.common.utils import now_in_utc
+
+from content_sync.tasks import sync_unsynced_websites
+from websites.constants import CONTENT_TYPE_RESOURCE
+from websites.models import WebsiteContent
+
+
+def write_to_csv(self, path: str, modified_content: List[Dict]):
+    """Write modified contents to csv."""
+
+    with open(path, "w", newline="") as csvfile:
+        if not modified_content:
+            return
+        fieldnames = modified_content[0].keys()
+        self.stdout.write(f"{fieldnames}")
+        writer = csv.DictWriter(csvfile, fieldnames, quoting=csv.QUOTE_ALL)
+        writer.writeheader()
+        for content in modified_content:
+            writer.writerow(content)
+
+
+class Command(BaseCommand):
+    """ Fix WebsiteContent files that are missing the Website name in their paths"""
+
+    help = __doc__
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-o",
+            "--out",
+            dest="out",
+            default=None,
+            help="If provided, a CSV file of WebsiteContent objects with modified paths will be written.",
+        )
+        parser.add_argument(
+            "-c",
+            "--commit",
+            dest="commit",
+            action="store_true",
+            default=False,
+            help="Whether the changes to WebsiteContent file paths should be saved to the database/backend.",
+        )
+        parser.add_argument(
+            "-p",
+            "--prefix",
+            dest="prefix",
+            default=None,
+            help="Prefix string for WebsiteContent query",
+            required=True,
+        )
+        parser.add_argument(
```

Comment:
The arguments are a bit confusing. Maybe we have  --fix to update the database/backend and --sync to run sync_unsynced_websites task?